### PR TITLE
Suppress sqlExceptionError logs from console

### LINKED ISSUE #998: Suppress duplicate Key value error message
Timestamp already exists errors - [Detail: Key (experiment_name, interval_end_time, interval_start_time)=(11787264|345199|683cb481-8f8f-4948-9173-29f730c5c03f|btr-spm-solution-prod|deployment|app-s2m-prd, 2023-10-11 00:30:00, 2023-10-11 00:15:01) already exists. 2023-10-1815:07:41.847 ERROR [qtp88397182-34][SqlExceptionHelper.java(138)]-ERROR: duplicate key value violates unique constraint "kruize_results_20231011_pkey" - This is due to the new cost operator that now sends aggregated records through out the day for customer that have enabled it.

For the last 6 hr period of the day, Kruize will have to reject 75% of the records as these would already have been added. Kruize will for now suppress printing the “duplicate key value” error message.

### PR DESCRIPTION
**Pull request by khansaad**
This PR fixes #998

### CODE REVIEW

**Comment by chandrams** [comment]
@khansaad - Have you run the functional test suite?

**Comment by khansaad** [comment]
> @khansaad - Have you run the functional test suite?

Yes, attached result of the same:

```
~~~~~~~~~~~~~~~~~~~~~~~~~~~~ Overall summary of the tests ~~~~~~~~~~~~~~~~~~~~~~~
Total time taken to perform the test 1507 seconds
Total Number of test suites performed 1
Total Number of tests performed 350
Total Number of tests passed 161
Total Number of tests failed 189

```
[Uploading functional_tests.zip…]()

File: src/main/java/com/autotune/Autotune.java [hunk 1]
```diff
@@ -59,6 +61,9 @@ public class Autotune {
 
     public static void main(String[] args) {
 
+        // Turning off the logging level for the specific package to reduce console logging
```
Comment (msvinaykumar) [inline src/main/java/com/autotune/Autotune.java:64]: Include a try and catch block. If we transition to a different package, this action may result in an error.

File: src/main/java/com/autotune/analyzer/exceptions/KruizeErrorHandler.java [hunk 2]
```diff
@@ -59,12 +59,25 @@ public void handle(String target, Request baseRequest, HttpServletRequest reques
...
+                } else {
+                    LOGGER.error(gsonStr);
+                }
+            } else {
+                myList.forEach(failedResult ->
+                        failedResult.getErrors().forEach(error -> {
+                            if (error.getHttpcode() == 409) {
+                                LOGGER.debug(gsonObj.toJson(error));
+                            } else {
+                                LOGGER.error(gsonObj.toJson(error));
```
Comment (msvinaykumar) [inline src/main/java/com/autotune/analyzer/exceptions/KruizeErrorHandler.java:76]: Ensure error not printing twice

File: src/main/java/com/autotune/operator/InitializeDeployment.java [hunk 3]
```diff
@@ -111,7 +111,7 @@ else if (deploymentInfoField.getType() == Integer.class) {
                 } else
                     throw new IllegalAccessException("Failed to set " + deploymentInfoField + "due to its type " + deploymentInfoField.getType());
             } catch (Exception e) {
-                LOGGER.warn("Error while setting config variables : {} : {}", e.getClass(), e.getMessage());
+                LOGGER.debug("Error while setting config variables : {} : {}", e.getClass(), e.getMessage());
```
Comment (msvinaykumar) [inline src/main/java/com/autotune/operator/InitializeDeployment.java:114]: let it be warn it self,  but add this  
```
if (null != deploymentInfoFieldValue) {
                    if (deploymentInfoField.getType() == String.class)
                        deploymentInfoField.set(null, deploymentInfoFieldValue);
                    else if (deploymentInfoField.getType() == Boolean.class)
                        deploymentInfoField.set(null, Boolean.parseBoolean(deploymentInfoFieldValue));
                    else if (deploymentInfoField.getType() == Integer.class) {
                        deploymentInfoField.set(null, Integer.parseInt(deploymentInfoFieldValue));
                    } else
                        throw new IllegalAccessException("Failed to set " + deploymentInfoField + "due to its type " + deploymentInfoField.getType());
                }


```

**Comment by khansaad** [comment]
> @khansaad - Have you run the functional test suite?

Re-Ran the tests after merging the latest code, now the records are matching with 21 rm image

```
~~~~~~~~~~~~~~~~~~~~~~~~~~~~ Overall summary of the tests ~~~~~~~~~~~~~~~~~~~~~~~
Total time taken to perform the test 3029 seconds
Total Number of test suites performed 1
Total Number of tests performed 350
Total Number of tests passed 236
Total Number of tests failed 114
```
[remote_monitoring_tests.zip](https://github.com/kruize/autotune/files/13600160/remote_monitoring_tests.zip)

**Review by dinogun — APPROVED** [review]
LGTM

**Pull request merged**